app/models/miq_expression.rb

```diff
@@ -1026,22 +1026,22 @@ def self.evaluate(expression, obj, _inputs = {}, tz = nil)
     result
   end
 
-  def evaluate(obj, inputs = {}, tz = nil)
-    self.class.evaluate(self, obj, inputs, tz)
+  def evaluate(obj, tz = nil)
+    self.class.send(:evaluate, self, obj, tz)
```

Comment:
This is now violating `.evaluate`'s privacy....WDYT about dropping the `private_class_method` for now?